### ISSUE TITLE
Looking at other farm  parcels in the region

### ISSUE THREAD

**Issue by NicolaJHarvey** [dmnfarrell/tracebtb#13]
Does the all parcels in the region work?  Can a distance (radius from index farm) be set when deciding which land parcels to include?

**Comment by dmnfarrell** [comment]
you mean look at neighbouring parcels? this works but requires the LPIS master file to be loaded first. it will display all parcels in the region of the map being viewed. but maybe a radius would be better.

**Comment by NicolaJHarvey** [comment]
Yes looking at neighbouring parcels is what I mean. A radius of 50m is used in DAFM to assess contiguous herds and a buffer of 2km and 5km in the wildlife maps.  So something like that would be useful.
